include/mbedtls/mbedtls_config.h

```diff
@@ -3760,6 +3760,9 @@
  */
 //#define MBEDTLS_PSA_KEY_SLOT_COUNT 32
 
+/* RSA OPTIONS */
+#define MBEDTLS_RSA_GEN_KEY_MIN_SIZE            1024 /**<  Minimum RSA key size allowed in bits (Minimum possible value is 128 bits) */
```

Comment:
I see in PSA we have `PSA_MAX_KEY_BITS`, so would this be better (more explicit) as `MBEDTLS_RSA_GEN_KEY_MIN_BITS`?